react/features/display-name/components/native/DisplayNameLabel.tsx

```diff
@@ -1,43 +1,39 @@
-// @flow
-
-import React, { Component } from 'react';
+import * as React from 'react';
 import { Text, View } from 'react-native';
 
 import {
     getParticipantById,
     getParticipantDisplayName
-} from '../../../base/participants';
-import { connect } from '../../../base/redux';
+} from '../../../base/participants/functions.js';
+import { connect } from '../../../base/redux/functions.js';
 
 import styles from './styles';
 
-type Props = {
+/**
+ * Renders a label with the display name of the on-stage participant.
+ */
+class DisplayNameLabel extends React.Component<{
```

Comment:
Let's please create a Props type definition, similar to what was there before, but TS. It should make the code changes smaller and IMHO it's a simpler way forward.

Reply:
Updated